src/Xunit.NetCore.Extensions/UseInvariantCultureAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Threading;
+using Xunit.Sdk;
+
+namespace Xunit.NetCore.Extensions
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
+    public class UseInvariantCultureAttribute : BeforeAfterTestAttribute
+    {
+        private CultureInfo originalCulture = CultureInfo.CurrentCulture;
```

Comment:
nit: I'd make it nullable instead of initializing with unused value
but for testing code I don't care much